lms/djangoapps/teams/tests/test_views.py

```diff
@@ -186,7 +202,7 @@ def make_call(self, url, expected_status=200, method='get', data=None, content_t
 
         If a user is specified in kwargs, that user is first logged in.
         """
-        user = kwargs.pop('user', 'student_enrolled')
+        user = kwargs.pop('user', 'student_enrolled_not_on_team')
```

Comment:
@explorerleslie I suspect that any such logic would be covered by the Create Team PR that @dan-f  has been reviewing.  We should check in with him to see if that is already being covered in the UI.